packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/SuggestionsStore.ts

```diff
@@ -40,14 +44,17 @@ export class SuggestionsStore<T> {
   @autobind
   private _ensureSuggestionModel(suggestion: ISuggestionModel<T> | T): ISuggestionModel<T> {
     if (this._isSuggestionModel(suggestion)) {
-      return suggestion as ISuggestionModel<T>;
+      return suggestion;
     } else {
       return {
         item: suggestion,
         selected: false,
         // tslint:disable-next-line:no-any
-        ariaLabel: (<any>suggestion).name || (<any>suggestion).primaryText
-      } as ISuggestionModel<T>;
+        ariaLabel:
+          this.getAriaLabel !== undefined
+            ? this.getAriaLabel(suggestion)
+            : ((suggestion as unknown) as ITag).name || (<IPersonaProps>suggestion).primaryText
```

Comment:
@JasonGore might be interested in the above typings.

Also we can prob now remove:

https://github.com/OfficeDev/office-ui-fabric-react/pull/8938/files#diff-e5fa1b52bbc709cb3ceae476ac81a466L48